### PR TITLE
Add simple snippets doc

### PR DESCRIPTION
**Pull request by TheCadien**
| Q | A
| --- | ---
| Fixed tickets | #460
| License | MIT

#### What's in this PR?

Add simple Snippet Doc

#### Why?

Doc is missing

### CODE REVIEW

File: book/snippets.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            </property>
+        </properties>
+    </template>
+
+Properties
+----------
+
+Properties are the same as Page :doc:`templates`.
+
+Load Snippets from a Subfolder
```
Comment (alexander-schranz) [inline book/snippets.rst]: As this is very uncommen to be changed. This should be moved into an own PR and should be added here about documenting how to change `page` and `snippet` folder directories:

https://docs.sulu.io/en/2.0/cookbook/override-dir-structure.html

File: book/snippets.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+By the means of configuration in `config/packages/sulu_admin.yaml` according to the following scheme
+it is also possible to load snippet templates from custom folders.
+
+.. code-block:: yaml
+
+    sulu_core:
+        content:
+            structure:
+                paths:
+                    events:
```
Comment (alexander-schranz) [inline book/snippets.rst]: a key like `event_snippets` would make more sense to not conflict when somebody has an `events` entity.

File: book/snippets.rst (hunk 1 above)
Comment (TheCadien) [reply book/snippets.rst]: ah, haven't seen the page at all.  That makes sense, I remove it and add it there again.

File: book/snippets.rst (hunk 2 above)
Comment (TheCadien) [reply book/snippets.rst]: Change it at new pr

File: book/index.rst [hunk 3]
```diff
@@ -16,6 +16,7 @@ Be sure to know a thing or two about `Symfony`_. Sulu is based on Symfony and st
     twig
     content-architecture
     webspaces
+    snippets
```
Comment (niklasnatter) [inline book/index.rst]: I think I would not add an additional chapter in the book, because a lot of projects dont need to use snippets. Instead i would structure it like this:

1. Add a section below the `Content Blocks` section in the templates chapter: https://docs.sulu.io/en/2.2/book/templates.html#content-blocks
2. Shortly explain snippets and its usecases in the section and add a link to the`SnippetBundle` documentation
3. Create the `SnippetBundle` documentation (similar to the `LocationBundle`) https://docs.sulu.io/en/latest/bundles/location.html 

What do you think about this? 🙂

File: book/snippets.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
+Using Snippets
```
Comment (niklasnatter) [inline book/snippets.rst]: would use this file as `SnippetBundle` documentation (similar to `LocationBundle` or `RouteBundle`)

**Comment by nnatter** [comment]
@TheCadien Like this - thanks a lot for your time! 🙂

File: book/index.rst (hunk 3 above)
Comment (TheCadien) [reply book/index.rst]: See your point. 

I generally wanted to expand on the snippets a bit more. 
However, I have decided for time reasons to create a first basic version as PR and add the rest after. 

Therefore the decision of a separate chapter.

but im interestet in your opinion, because of me, we can also gladly transfer the content to another chapter. :) !

Comment (niklasnatter) [reply book/index.rst]: yeah, i think it would make sense to create a `SnippetBundle` documentation similar to the `LocationBundle` for this 🙂

**Comment by TheCadien** [comment]
@nnatter I have changed the doc to the SnippetBundle :)

**Comment by niklasnatter** [comment]
@TheCadien Thanks a lot! 🙂
